public/app/features/explore/QueryLibrary/QueryLibraryContext.tsx

```diff
@@ -19,17 +17,33 @@ export type QueryLibraryContextType = {
    * @param datasourceFilters Data source names that will be used for initial filter in the library.
    * @param queryActionButton Action button will be shown in the library next to the query and can implement context
    *   specific actions with the library, like running the query or updating some query in the current app.
+   * @param options.context Used for tracking. Should identify the context this is called from, like 'explore' or
+   *   'dashboard'.
    */
-  openDrawer: (datasourceFilters: string[], queryActionButton: QueryActionButton) => void;
+  openDrawer: (
+    datasourceFilters: string[],
+    queryActionButton: QueryActionButton,
+    options?: { context?: string }
```

Comment:
As we have more places where library will be used it seemed more reasonable to just pass this as an arg for the tracking funcitons.